code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -173,3 +173,23 @@
 
 	R.emag_items = 1
 	return 1
+
+/obj/item/borg/upgrade/language
+	name = "Language module"
```

Comment:
Should be lowercase.

Reply:
I'm apparently very bad at noticing capitalization. Probably should be lowercase, yeah.